pymc_marketing/clv/__init__.py

```diff
@@ -1,12 +1,13 @@
-from pymc_marketing.clv import distributions, models
-from pymc_marketing.clv.models.beta_geo import BetaGeoModel
-from pymc_marketing.clv.models.gamma_gamma import (
+from pymc_marketing.clv.models import (
+    BetaGeoModel,
     GammaGammaModel,
     GammaGammaModelIndividual,
 )
+from pymc_marketing.clv.utils import customer_lifetime_value
 
 __all__ = (
     "BetaGeoModel",
     "GammaGammaModel",
     "GammaGammaModelIndividual",
+    "customer_lifetime_value",
```

Comment:
Is it intended for users to call this function via `GammaGammaModel`, or as a separate utility function? In [btyd](https://github.com/ColtAllen/btyd/blob/main/btyd/models/gamma_gamma_model.py) I eliminated this redundancy and made it an internal method of the model.

Reply:
> The `GammaGamma` model provides a wrapper method as well. This is more general in that you could theoretically use a model other than the GammaGamma to estimate the monetary value.

I like the flexibility of this approach. Google has a guide on building a DL variant of the Gamma-Gamma model:

https://cloud.google.com/ai-platform/docs/clv-prediction-with-offline-training-train

Reply:
> We can also make it just part of the CLV models that estimate monetary_value, maybe as a Mixin

I definitely want to explore continuous-time CLV calculations in a future PR, as we've discussed before:

https://github.com/ColtAllen/btyd/issues/76

The calculation itself is specific to the model predicting expected transactions (such as the BG/NBD or Pareto/NBD), because the expected future transactions themselves are discounted to a present value. However, it'd be easy enough to callback this function via the Gamma-Gamma model.